validator/validator.go

```diff
@@ -30,11 +29,40 @@ func Validator() *validator.Validate {
 			panic("not a felt")
 		}, felt.Felt{}, &felt.Felt{})
 		v.RegisterCustomTypeFunc(func(field reflect.Value) any {
-			if t, ok := field.Interface().(rpc.TransactionType); ok {
+			if t, ok := field.Interface().(TransactionType); ok {
 				return t.String()
 			}
 			panic("not a TransactionType")
-		}, rpc.TransactionType(0))
+		}, TransactionType(0))
 	})
 	return v
 }
+
+// TransactionType is duplicated here to prevent import cycles
```

Comment:
I think we should be able to avoid doing this. Can you share the import cycle that is hapenning?

Reply:
Sure, it just arises if you use rpc.TransactionType in validator.go. For example, validator imports rpc, which imports utils (eg class.go), which imports validator (network.go) which imports validator

Reply:
Later commit resolves this. Moving (cn NetworkCustom) Validate() to config.GetNetwork() stops the import cycle.